integration-tests/graal-incubating/src/test/java/io/opentelemetry/integrationtests/graal/IncubatingApiTests.java

```diff
@@ -23,4 +42,63 @@ void incubatingApiIsLoadedViaReflection() {
     assertThat(MeterProvider.noop().get("test").counterBuilder("test"))
         .isInstanceOf(ExtendedLongCounterBuilder.class);
   }
+
+  @Test
+  void incubatingLogSdk() {
```

Comment:
Not sure why, but graal `nativeTest` works without any additional `reflect-config.json` entries. 🤔